test_runner/src/main/kotlin/ftl/json/SavedMatrix.kt

```diff
@@ -27,7 +29,7 @@ class SavedMatrix(matrix: TestMatrix) {
 
     var billableVirtualMinutes: Long = 0
         private set
-    var billablePhysicalMinutes: Long = 0
+    var billablePhysicalSecondsMinutes: Long = 0
```

Comment:
could we think about other name, because I am confused if we use seconds or minutes

Reply:
Yeah, definitely. We have yet another round of refactor ahead of us

Reply:
Thanks, its my fail. Its should be billablePhysicalMinutes